src/batch-client/index.ts

```diff
@@ -1083,10 +1084,15 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		}).then(res => {
 			let prefs: any = mapValuesDeep(res._attrs, coerceStringToBoolean);
-			prefs.zimbraPrefMailTrustedSenderList =
-				typeof prefs.zimbraPrefMailTrustedSenderList === 'string'
-					? castArray(prefs.zimbraPrefMailTrustedSenderList)
-					: prefs.zimbraPrefMailTrustedSenderList;
+
+			Object.keys(prefs).map(pref => {
```

Comment:
If there is only the need of iteration of the array, we can use forEach instead of map.
And we can also skip Object.keys() by using for...in loop for the object.

Reply:
Object is object not array

Reply:
updated